op-e2e/gas_price_oracle_test.go

```diff
@@ -19,7 +22,44 @@ import (
 	"github.com/ethereum-optimism/optimism/op-bindings/predeploys"
 )
 
+func inputsToHex(inputs []interface{}) []byte {
+	resultBytes := []byte{}
+	for _, input := range inputs {
+		switch v := input.(type) {
+		case int32:
+			bytes := make([]byte, 4)
+			binary.BigEndian.PutUint32(bytes, uint32(v))
+			resultBytes = append(resultBytes, bytes...)
+		case uint32:
+			bytes := make([]byte, 4)
+			binary.BigEndian.PutUint32(bytes, v)
+			resultBytes = append(resultBytes, bytes...)
+		case uint64:
+			bytes := make([]byte, 8)
+			binary.BigEndian.PutUint64(bytes, v)
+			resultBytes = append(resultBytes, bytes...)
+		default:
+			fmt.Errorf("unsupported type: %T", v)
```

Comment:
this should be returned from the function... the return value signature should be `([]byte, error)`